console/console-init/ui/src/modules/iot-device/components/ViewMetadata/MetadataReviewRows.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import { MetadataReviewRow } from "./MetadataReviewRow";
+import { DataType } from "constant";
+interface IMetadataReviewRowProps {
+  values: any[];
+  prevkey?: string;
+}
+
+const MetadataReviewRows: React.FC<IMetadataReviewRowProps> = ({
+  values,
+  prevkey
+}) => {
+  return (
+    <>
+      {values.map((val, index) => (
+        <>
```

Comment:
Use <Fragment> with a key as jest throws a warning for cases key-less fragments.